tensorflow/core/common_runtime/next_pluggable_device/next_pluggable_device.cc

```diff
@@ -23,8 +23,10 @@ limitations under the License.
 #include "tensorflow/core/common_runtime/dma_helper.h"
 #include "tensorflow/core/common_runtime/next_pluggable_device/next_pluggable_device_allocator.h"
 #include "tensorflow/core/framework/tensor.pb.h"
+#include "tensorflow/core/framework/variant_op_registry.h"
 #include "tensorflow/core/platform/errors.h"
 #include "tensorflow/core/platform/status.h"
+#include "tensorflow/core/util/reffed_status_callback.h"
```

Comment:
does the BUILD need to be updated?

Reply:
I didn't met build issue, it is in framework_internal_private_hdrs, which is depended by core:framework, it is in dependence chain of nextpluggabledevice, so I think it should be ok.  Does it caused build failure? thanks

Reply:
Verified that there is no build issue. Thanks!